yt_dlp/extractor/neteasemusic.py

```diff
@@ -155,6 +159,9 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'timestamp': 1202745600,
             'duration': 263,
             'thumbnail': r're:^http.*\.jpg',
+            'album': 'Piano Solos Vol. 2',
+            'album_artist': '',
```

Comment:
how is this `album_artist` value an empty string when the extraction code ends with `or None`?

Reply:
> how is this `album_artist` value an empty string when the extraction code ends with `or None`?

fixed